flask_cors/__init__.py

```diff
@@ -238,17 +244,25 @@ def cors_after_request(resp):
                 The actual after-request handler, retains references to the
                 the options, and definitions of resources through a closure.
             '''
+            logger.debug('Checking CORS for request')
             # If CORS headers are set in a view decorator, pass
             if resp.headers.get(ACL_ORIGIN):
+                logger.debug('Header {} have been set already, skipping'.format(ACL_ORIGIN))
                 return resp
 
+            logger.debug('Checking available CORS rules')
             for res_regex, res_options in resources:
                 if _try_match(res_regex, request.path):
+                    logger.debug('Rule {} matches'.format(str(res_regex)))
                     _options = options.copy()
                     _options.update(res_options)
                     _serialize_options(_options)
+                    logger.debug('Final CORS options: {}'.format(str(_options)))
                     _set_cors_headers(resp, _options)
                     break
+            else:
+                logger.debug('No CORS rule matches')
+            logger.debug('Response headers: {}'.format(str(resp.headers)))

```

Comment:
I think this log statement is duplicated when we actually set the headers. I vote to remove this one, and keep the other